govuk_chat_evaluation/rag_answers/data_models.py

```diff
@@ -161,10 +161,11 @@ def metric_instances(self):
 class RunMetricOutput:
     run: int
     metric: str
-    score: float
+    score: float | None = None
     cost: float | None = None
     reason: str | None = None
     success: bool | None = None
+    error: str | None = None
```

Comment:
Very minor - but it seems a more common convention to define these as `Optional[type] = None` rather than a type union

Reply:
`str | None = None` is the recommended syntax for Python 3.10+, which we are using 
https://peps.python.org/pep-0604/
https://discuss.python.org/t/clarification-for-pep-604-is-foo-int-none-to-replace-all-use-of-foo-optional-int/26945

albeit `Optional` is still valid, the `|` is preferred for 3.10+

Reply:
Thanks! I didn't know that. I can't really see from the link though that there is a stance that one is preferred over the other, seems like lots of debate - I expected more resolve to decide on one true way from the Python community 😝

It's a shame ruff hasn't got a view of it, at least for our configuration. I'll not comment on this one further